astropy/cosmology/flrw/base.py

```diff
@@ -269,7 +268,12 @@ def __post_init__(self):
     def Ob0(self, param, value):
         """Validate baryon density to None or positive float > matter density."""
         if value is None:
-            return value
+            warnings.warn(
+                "Ob0=None is deprecated, use Ob0=0 instead.",
```

Comment:
Also mention that there will no failures!

Reply:
✅